lib/agenda.js

```diff
@@ -36,7 +38,50 @@ WHERE {
   return null;
 }
 
+async function getAgendasForSubcase(subcaseId, useSudo=false) {
+  const queryString = `PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX besluitvorming: <https://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+
+SELECT DISTINCT (?meeting AS ?uri) ?meetingId ?agendaId ?agendaitemId ?plannedStart ?status ?kind ?number
+${useSudo ? `FROM ${sparqlEscapeUri(GRAPHS.KANSELARIJ)}` : ''}
+WHERE {
+  ?subcase a dossier:Procedurestap ;
+    mu:uuid ${sparqlEscapeString(subcaseId)} ;
+    ^besluitvorming:vindtPlaatsTijdens ?agendaActivity .
+
+  ?agendaActivity besluitvorming:genereertAgendapunt ?agendaitem .
+  
+  ?agendaitem mu:uuid ?agendaitemId .
+
+  ?agenda dct:hasPart ?agendaitem ;
+    mu:uuid ?agendaId ;
+    besluitvorming:isAgendaVoor ?meeting ;
+    besluitvorming:agendaStatus ?status .
+  FILTER NOT EXISTS { ?newer prov:wasRevisionOf ?agenda }
```

Comment:
this FILTER NOT EXISTS may be causing links not to be generated when they should be.
The `prov:wasRevisionOf` is  already propagated, the actual agenda data is not.
So we correctly saying "You cannot see design agenda B"
But we are not saying "You should however be able to see approved agenda A since subcase is also on that one"

Reply:
this could fix the "intern-overheid" issue.
`FILTER NOT EXISTS { ?newerId ^mu:uuid/prov:wasRevisionOf ?agenda }`
So we don't only check the already propagated relation (if it exists) but also if it exists if the id was propagated

This results in non-sudo queries getting the correct "latest agenda" they should be able to see on their graph.
It does not fix the sudo query for "dossierbeheerder". they still get designAgenda B as "not visible"
So no link for them. Thinking of how to fix that and also keep it working for the entire postpone/resubmit process.